### ISSUE TITLE
UI papercuts

### ISSUE THREAD

**Issue by bertob** [TenderOwl/Frog#50]
A few minor UI issues I came across testing the app

- The icon and text should be centered together on the empty state buttons

![image](https://user-images.githubusercontent.com/1908896/168685030-51451fbc-abb0-4c74-8d4c-91c7c7153608.png)

- The action bar should have a divider line above
- The buttons in the action bar should not be raised, i.e. have no background unless hovered/active
- The "take a shot" button should probably have the `.suggested-action` class. Also, its label should use title casing, and I'd make it more explicit, "shot" is a bit vague. How about "Take a Screenshot" or "New Scan"?

![image](https://user-images.githubusercontent.com/1908896/168685255-2bf81b6c-6243-4e47-8754-36f9d0d9c233.png)

- The "Configure" menu item could be more explicit, e.g. "Manage Languages"

**Comment by jannuary** [comment]
While we're at it, the Language list is quite ugly, and the scrollbar is in the wrong spot:
![image](https://user-images.githubusercontent.com/27908024/171992075-41e7fb77-591d-4bb3-b79c-28d31a94d8f7.png)
I'd remove the padding from the main box, putting it on the list instead, set ```selection-mode: none``` on it, make the download buttons the activatable widgets and used AdwClampScrollable instead of AdwClamp. In addition, .flat isn't really supposed to be used with scrolled views so it intersects the list nastily.

**Comment by amka** [comment]
> While we're at it, the Language list is quite ugly, and the scrollbar is in the wrong spot: ![image](https://user-images.githubusercontent.com/27908024/171992075-41e7fb77-591d-4bb3-b79c-28d31a94d8f7.png) I'd remove the padding from the main box, putting it on the list instead, set `selection-mode: none` on it, make the download buttons the activatable widgets and used AdwClampScrollable instead of AdwClamp. In addition, .flat isn't really supposed to be used with scrolled views so it intersects the list nastily.

Your idea is a good one. But I have to mention this window will be totally redesigned in near future. I'll split installed and available languages, and add more options to configure.

**Comment by amka** [comment]
@bertob Thank you! I hope most of the point were fixed in 1.1.0 release.

**Comment by bertob** [comment]
Nice, yeah the points from the original issue all look fixed to me!

I agree with @jannuary's suggestions about the language dialog, but maybe that should be a separate issue if it'll be redesigned soon?

**Comment by amka** [comment]
Yep, since you think your issue is done, I'll move this one to its own issue and plan it for the next release.
